component/src/test/java/com/celements/calendar/CalendarTest.java

```diff
@@ -249,11 +248,11 @@ public void testGetDocumentReference() {
 
   @Test
   public void testGetStartDate_newDate() {
-    java.util.Calendar gregCal = GregorianCalendar.getInstance();
+    java.util.Calendar gregCal = java.util.Calendar.getInstance();
```

Comment:
"Using the static factory method Calendar.getInstance() isn't a recommended approach as it will return an instance subjective to the default locale."
https://www.baeldung.com/java-gregorian-calendar
please change back to GregorianCalendar